Jenkinsfile.conda

```diff
@@ -0,0 +1,2 @@
+@Library('JenkinsShared')_
+CondaPipeline([], "ts-dateloc", "lsst.ts.dateloc")
```

Comment:
I see that in `conda/meta.yaml` you specified `noarch` so you need to add that here as well like so:

`CondaPipeline([], "ts-dateloc", "lsst.ts.dateloc", "noarch")`